[2024-11-26T06:55:26.954+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-26T06:55:26.974+0000] {taskinstance.py:2602} INFO - Dependencies not met for <TaskInstance: extract_weather_dag.extract_data scheduled__2024-11-25T00:00:00+00:00 map_index=8 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-11-26T06:55:26.981+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
